lib/utils.js

```diff
@@ -38,19 +38,19 @@ export function focus(item: ?mixed) {
   }
 }
 
-export function openOrShowDock(URI: string) {
+export async function openOrShowDock(URI: string): Promise<any> {
   // atom.workspace.open(URI) will activate/focus the dock by default
   // dock.toggle() or dock.show() will leave focus wherever it was
 
   // this function is basically workspace.open, except it
-  // will not focus the pane if there is an open instance of that view
-
+  // will not focus the newly opened pane
   const dock = atom.workspace.paneContainerForURI(URI);
-  if (!dock) {
-    atom.workspace.open(URI, { searchAllPanes: true });
-  } else {
-    dock.show();
-  }
+  return dock
+    ? dock.show()
+    : atom.workspace.open(URI, {
+        searchAllPanes: true,
+        activatePane: false
+      });
 }
```

Comment:
The show part should be last so it opens if not already opened, then shows the dock workout focusing it. I'll rebase this PR later.